data/actionsModels.js

```diff
@@ -0,0 +1,36 @@
+const knex = require('knex');
+const dbConfig = require('../knexfile');
+const db = knex(dbConfig.development);
+
+function get(id){
+    if(id){
+        return db('actions').where('id', id).then(action => { return action[0] })
```

Comment:
The `then` here seems a bit lengthy; you could simplify it with an implicit return since you're already using an arrow function:
```javascript
return db('actions').where('id', id).then(action => action[0])
```
you could also use an object as the argument for `where`:
```javascript
return db('actions').where({ id: id }).then(action => action[0])
```
and then simplify it down more with shorthand object notation since the key and the variable for the value are the same:
```javascript
return db('actions').where({ id }).then(action => action[0])
```
the last thing I'm looking at is the callback function you pass into your `then`. You call the parameter `action`, when it should probably be `actions` since it's an array. Another option is just using the `first` method to do the same thing you're doing!
```javascript
return db('actions').where({ id }).first()
```